contents/blog/the-posthog-array-1-33-0.md

```diff
@@ -0,0 +1,124 @@
+---
+date: 2022-02-28
+title: Array 1.33.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Release notes", "Product updates"]
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+excerpt: Introducing a full multivariate Experimentation suite, granular permissions on dashboards, Altinity Cloud support for ClickHouse, configuring your instance from the PostHog UI and 290+ improvements/fixes.
+---
+
+PostHog 1.33.0 introduces a full Experimentation suite! A/B/C/... changes to your product to ensure you maximize value for your end users. For Enterprise customers, you can now set granular permissions on dashboards. An easier way to configure your instance, support for Altinity Cloud, insight legends, insight permalinks and 290+ improvements/fixes are included.
+
+<blockquote class='warning-note'>
+<b>IMPORTANT!</b> Do not upgrade to this version if you have deployed PostHog using Postgres. PostHog no longer supports a Postgres-based installation (last version supported is <code>1.30.0</code>) and now requires Clickhouse. To use this version, you must <a href="/docs/self-host/migrate-from-postgres-to-clickhouse" target="_blank">upgrade to ClickHouse</a> first.
```

Comment:
This wording may be confusing because _everyone_ deploys PostHog using Postgres. We should be careful to indicate this warning is only intended for those who have an installation where _analytics_ is Postgres-based.